### PR TITLE
added religion test

### PR DESCRIPTION
**Pull request by ajpierskalla3**
figure it out

### CODE REVIEW

File: src/test/java/com/fb/faithfulblendzbe/ReligionRepositoryTests.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        assertNull(religion.getId());
+        religionRepository.save(religion);
+        assertNotNull(religion.getId());
+
+        Religion fetchedReligion = religionRepository.findById(religion.getId()).get();
+        assertEquals(religion.getId(), fetchedReligion.getId());
+    }
+
+    @Test
+    public void testDeleteOrganization() {
```
Comment (Jaden-Reklaw) [inline src/test/java/com/fb/faithfulblendzbe/ReligionRepositoryTests.java]: This needs to have a different name for the function

File: src/test/java/com/fb/faithfulblendzbe/ReligionRepositoryTests.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            Religion religion = new Religion();
+            religion.setReligion_name("religion"+i);
+            rel.add(religion);
+        });
+
+        religionRepository.saveAll(rel);
+        for (Religion religion : rel) {
+            assertNotNull(religion.getId());
+        }
+        Iterable<Religion> fetchedOrgs = religionRepository.findAll();
```
Comment (Jaden-Reklaw) [inline src/test/java/com/fb/faithfulblendzbe/ReligionRepositoryTests.java]: This naming of the variables needs to be changed

**Comment by ajpierskalla3** [comment]
If you see anything else let me know

**Comment by ajpierskalla3** [comment]
Did this work?

**Review by Jaden-Reklaw — APPROVED** [review]
Looks good all changes will be commited to main

**Pull request merged**